addons/survey/models/survey_survey.py

```diff
@@ -105,6 +107,25 @@ def _get_default_access_token(self):
     certification_give_badge = fields.Boolean('Give Badge')
     certification_badge_id = fields.Many2one('gamification.badge', 'Certification Badge')
     certification_badge_id_dummy = fields.Many2one(related='certification_badge_id', string='Certification Badge ')
+    # live sessions
+    session_state = fields.Selection([
+        ('not_started', 'Not Started'),
+        ('ready', 'Ready'),
+        ('in_progress', 'In Progress'),
+        ], default='not_started', string="Session State", required=True, copy=False)
+    # live sessions - current question fields
+    session_current_question_id = fields.Many2one('survey.question', string="Current Question", copy=False,
```

Comment:
Indeed, after checking, survey users can read all surveys but only modify their own.
I added a sudo'ed write in the "session_next_question" method to avoid ACL issues.
It should hopefully be sufficient.

Reply:
Currently just ensure user has survey_user group and ok for sudo for those fields or be sure for read (put a comment because if somehow we move some code implicit check on read could be forgotten).